package.json

```diff
@@ -72,7 +73,7 @@
         "semver": "^7.6.0"
     },
     "peerDependencies": {
-        "aws-cdk-lib": "2.133.0",
-        "aws-cdk": "2.133.0"
+        "aws-cdk-lib": "2.145.0",
```

Comment:
This requires a global replace in the project for all instance of 2.133 since the docs have this version specified and it is also used in codepipeline. 
Let's either remove this particular change and I will update it as part of the release (next week) or do it as a global replace.

Reply:
ok done